publications-2007.bib

```diff
@@ -229,6 +229,17 @@ @PhDThesis{2007:nigro:discontinuous
   year    = 2007
 }
 
+@Article{2007:nomura.sato.ea:multi-disciplinary,
+  author  = { Nomura, Tsuyoshi and Sato, Kazuo and Nishiwaki, Shinji and Yoshimura, Masataka },
+  title   = { Multi-disciplinary Multi-objective Topology Optimization of Electromagnetics and Structural Mechanics (For Case of Optimal Dielectric Resonator Antenna Designs) },
+  journal = { Trans. JSME, Ser.A },
+  year    = 2007,
+  volume  = 73,
+  pages   = { 1111--1119 },
+  issue   = 734,
```

Comment:
`issue` is not BibTeX canon. Standard syntax is `number`. See section 4.2 Field in the documentation
https://mirrors.mit.edu/CTAN/biblio/bibtex/base/btxdoc.pdf

I believe the issue field only exists in biblatex, not bibtex.

As a consequence, the `issue` field will not be shown in `jabref`, see
https://github.com/dealii/publication-list/blob/c9fbd395b1c179fc0f75da6876da7d152fc67d5c/jabref-template/listrefs.layout

Yes, bibtex really *is* confusing.